lib/ansible/modules/network/eos/eos_vrf.py

```diff
@@ -189,7 +189,7 @@ def map_obj_to_commands(updates, module):
 
 def map_config_to_obj(module):
     objs = []
-    output = run_commands(module, ['show vrf'])
+    output = run_commands(module, ['show vrf | text'])
```

Comment:
Rather than creating something not supported by the device which will inevitably confuse someone later, you should probably do `run_commands(module, {'command': 'show vrf', 'output': 'text'})`

Reply:
Yep.. makes sense, I have updated the code accordingly.